cli/lsp/language_server.rs

```diff
@@ -640,16 +670,11 @@ impl Inner {
     self.documents.open(
       specifier.clone(),
       params.text_document.version,
-      params.text_document.text,
+      params.text_document.text.clone(),
```

Comment:
I think you should be able to pass `&params.text_document.text` here if you change `open()` to take a `&str`. That method seems to be making a copy anyway.